### DISCUSSION TITLE
sliamb/prebuild-paopaodns和sliamb/paopaodns有什么区别；

### DISCUSSION THREAD

**Discussion by gukey** [kkkgo/PaoPaoDNS#206] [问答和帮助]
无意中看到这个 docker 镜像，但没找到具体的使用说明，能说说吗；

**Comment by kkkgo** [comment]
prebuild是docker多阶段构建的二进制中间产物，不是最终可用的镜像，预构建二进制大部分情况下可以节省构建时间。

**Reply by kkkgo** [reply]
具体构建过程可以参考源码的dockerfile。

**Reply by gukey** [reply]
好的，谢谢！
